### PR TITLE
Noah stoeckel

### PR DESCRIPTION
**Pull request by BoxOfCereal**
Pulling harder than I've ever pulled before.

### CODE REVIEW

File: src/controls/Controls.spec.js [hunk 1]
```diff
@@ -1 +1,61 @@
...
+    const toggleLockedMock = jest.fn();
+    //need to close it before locking
+    const { getByText } = render(
+      <Controls toggleLocked={toggleLockedMock} closed={true} />
+    );
+
+    const lockButton = getByText(/lock gate/i);
+    fireEvent.click(lockButton);
+
+    expect(toggleLockedMock).toBeCalledTimes(1);
```
Comment (johnoro) [inline src/controls/Controls.spec.js:50]: A following test could test the case that you click the lock gate button multiple times; what is supposed to happen after the first? I'd assume you'd want the callback to not be called at that point, but would it be? It's that kind of motivation that we usually have behind these tests; we want to get _all_ of the expected functionality tested, odd edge cases included (if the user could do it, we want to test it typically).

**Review by johnoro — APPROVED** [review]
Looking absolutely awesome!
